setup/ci/codebuild.md

```diff
@@ -18,7 +18,24 @@ The AWS Codebuild stage requires Spinnaker 1.19 or later.
 
 ### AWS CodeBuild project
 
-You need to have an [AWS CodeBuild](https://aws.amazon.com/codebuild/) project.
+You need to have an [AWS CodeBuild](https://aws.amazon.com/codebuild/) project. To create a project,
+follow [instructions](https://docs.aws.amazon.com/codebuild/latest/userguide/setting-up.html#setting-up-service-role-cli)
+to create a service role, then run the following command, replace the service role with the one created and region of your own choice
+
+```
+aws codebuild create-project \
+  --name spinnaker-project \
+  --source "type=GITHUB,location=https://github.com/aws/aws-codebuild-docker-images.git" \
+  --artifacts "type=NO_ARTIFACTS" \
```

Comment:
Can you add a note below about how this won't produce artifacts, so they need to follow some use-case sample to setup artifacts in the project in order to deploy something in the pipeline.

Reply:
Added.